### PR TITLE
Add Live Search Widgets (EDS Delivery)

### PR DESCRIPTION
**Pull request by herzog31**
This PR is using a custom build from https://github.com/adobe/storefront-product-listing-page/pull/13.

Test URLs:
- Before: https://main--aem-boilerplate-commerce--hlxsites.hlx.page/
- After: 
    - https://plp-widget-local--aem-boilerplate-commerce--hlxsites.hlx.live/
    - https://plp-widget-local--aem-boilerplate-commerce--hlxsites.hlx.live/gear
    - https://plp-widget-local--aem-boilerplate-commerce--hlxsites.hlx.live/search?q=watch

### CODE REVIEW

File: blocks/product-list-page/product-list-page.js [hunk 1]
```diff
@@ -1,321 +1,63 @@
...
+      minQueryLength: '2',
+      currencySymbol: '$',
+      currencyRate: '1',
+      displayOutOfStock: true,
+      allowAllProducts: false,
+      imageCarousel: false,
+      optimizeImages: true,
+      imageBaseWidth: 200,
+      listview: true,
+      displayMode: '', // "" for plp || "PAGE" for category/catalog
```
Comment (arumsey) [inline blocks/product-list-page/product-list-page.js:33]: When should a display mode of `PAGE` be used? Currently search.js will fetch all the commerce data but not render anything. Is that the expected behaviour? What is a `category/catalog` view meant to display?

Comment (herzog31) [reply blocks/product-list-page/product-list-page.js:33]: In Adobe Commerce Luma there are alternative display modes for category pages which display mostly banners. So most of the time display mode `PAGE` should not be used.

* `displayMode: 'PAGE'`: https://magento2-demo.magebit.com/women.html
* `displayMode: ''`: https://magento2-demo.magebit.com/women/tops-women/hoodies-and-sweatshirts-women.html

Comment (arumsey) [reply blocks/product-list-page/product-list-page.js:33]: 👍 So the `PAGE` display mode just hasn't been implemented yet in the storefront PLP project?

**Comment by arumsey** [comment]
@herzog31 Where did you obtain `scripts/widgets/LiveSearchAutocomplete.js` from?

**Pull request merged**